tests/integration/staking/keeper/determinstic_test.go

```diff
@@ -220,321 +175,361 @@ func (suite *DeterministicTestSuite) getStaticValidator2() stakingtypes.Validato
 		DelegatorAddr: delegator1,
 		ValidatorAddr: validator.OperatorAddress,
 	}
 
-	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.DelegatorValidator, 3563, false)
+	testdata.DeterministicIterations(f.ctx, tt, req, f.queryClient.DelegatorValidator, 3563, false)
 }
 
-func (suite *DeterministicTestSuite) TestGRPCDelegatorUnbondingDelegations() {
-	rapid.Check(suite.T(), func(t *rapid.T) {
-		numVals := rapid.IntRange(1, 5).Draw(t, "num-vals")
-		delegator := testdata.AddressGenerator(t).Draw(t, "delegator")
+func TestGRPCDelegatorUnbondingDelegations(t *testing.T) {
+	t.Parallel()
+	f := initFixture(t)
+
+	tt := require.New(t)
+	rapid.Check(t, func(rt *rapid.T) {
+		numVals := rapid.IntRange(1, 5).Draw(rt, "num-vals")
+		delegator := testdata.AddressGenerator(rt).Draw(rt, "delegator")
 
 		for i := 0; i < numVals; i++ {
-			validator := suite.createAndSetValidatorWithStatus(t, stakingtypes.Bonded)
-			shares, err := suite.createDelegationAndDelegate(t, delegator, validator)
-			suite.Require().NoError(err)
+			validator := createAndSetValidatorWithStatus(rt, f, t, stakingtypes.Bonded)
+			shares, err := createDelegationAndDelegate(rt, f, t, delegator, validator)
+			assert.NilError(t, err)
 
-			_, err = suite.stakingKeeper.Undelegate(suite.ctx, delegator, validator.GetOperator(), shares)
-			suite.Require().NoError(err)
+			_, err = f.app.StakingKeeper.Undelegate(f.ctx, delegator, validator.GetOperator(), shares)
+			assert.NilError(t, err)
 		}
 
 		req := &stakingtypes.QueryDelegatorUnbondingDelegationsRequest{
 			DelegatorAddr: delegator.String(),
-			Pagination:    testdata.PaginationGenerator(t, uint64(numVals)).Draw(t, "pagination"),
+			Pagination:    testdata.PaginationGenerator(rt, uint64(numVals)).Draw(rt, "pagination"),
 		}
 
-		testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.DelegatorUnbondingDelegations, 0, true)
+		testdata.DeterministicIterations(f.ctx, tt, req, f.queryClient.DelegatorUnbondingDelegations, 0, true)
 	})
 
-	suite.SetupTest() // reset
+	initFixture(t) // reset
 
-	validator := suite.getStaticValidator()
-	shares1, err := suite.fundAccountAndDelegate(delegatorAddr1, validator, suite.amt1)
-	suite.Require().NoError(err)
+	validator := getStaticValidator(f, t)
+	shares1, err := fundAccountAndDelegate(f, t, delegatorAddr1, validator, f.amt1)
+	assert.NilError(t, err)
 
-	_, err = suite.stakingKeeper.Undelegate(suite.ctx, delegatorAddr1, validatorAddr1, shares1)
-	suite.Require().NoError(err)
+	_, err = f.app.StakingKeeper.Undelegate(f.ctx, delegatorAddr1, validatorAddr1, shares1)
+	assert.NilError(t, err)
 
 	req := &stakingtypes.QueryDelegatorUnbondingDelegationsRequest{
 		DelegatorAddr: delegator1,
 	}
 
-	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.DelegatorUnbondingDelegations, 1302, false)
+	testdata.DeterministicIterations(f.ctx, tt, req, f.queryClient.DelegatorUnbondingDelegations, 1308, false)
```

Comment:
Curious, why this value changed?

Reply:
May be L507 should be the reason, try
```
f = initFuxture(t)
```
May be the same thing continued in all tests
@likhita-809